lang/typescript/README.md

```diff
@@ -1,6 +1,34 @@
 # Worldwide - Typescript
 
-🚧 Still in development -- not ready for use yet 🚧
+Utilities for parsing and formatting address fields
+
+## Usage
+
+```ts
+import {generateAddress1, generateAddress2} from '@shopify/worldwide';
+
+// Generate Address1
+generateAddress1({
+  countryCode: 'BR',
+  streetName: 'Main',
```

Comment:
Nit: can we use a more Brazilian sounding street name?

Reply:
I took most of the tests/examples from the ruby code which just uses "Main", got a better example?